src/pages/Options/components/GitHubToken.tsx

```diff
@@ -20,6 +20,54 @@ const GitHubToken = () => {
     fetchToken();
   }, []);
 
+  const handleOAuthToken = async () => {
+    const clientId = 'Ov23liyofMsuQYwtfGLb';
+    const redirectUri = chrome.identity.getRedirectURL();
+    const scope = encodeURIComponent('read:user');
+    const authUrl = `https://github.com/login/oauth/authorize?client_id=${clientId}&redirect_uri=${redirectUri}&scope=${scope}&response_type=token`;
+
+    console.log('[FastPR]: Start authorization...');
+    chrome.identity.launchWebAuthFlow(
+      {
+        url: authUrl,
+        interactive: true,
+      },
+      function (redirectUrl) {
+        if (chrome.runtime.lastError || !redirectUrl) {
+          console.error(chrome.runtime.lastError ? chrome.runtime.lastError.message : 'Authorization failed.');
+          return;
+        }
+        const code = new URL(redirectUrl).searchParams.get('code');
+        console.log('[FastPR]: Get session code:', code?.slice(0, 2).padEnd(code.length - 2, '*'));
+        fetch('http://8.147.129.123/github', {
```

Comment:
This URL will be replaced as `https://oauth.hypercrx.cn/github` in the future after the domain is valid in few days.